### ISSUE TITLE
When copying from CPU, table comes cropped

### ISSUE THREAD

**Issue by fred26** [x64dbg/x64dbg#1444]
Is there a way to copy the entire table contents from each column of the CPU? The size of the copied columns dependes on the width of the columns and the width of them are limited, it does not depend on the size of the text itself.

Thanks

**Comment by mrexodia** [comment]
No, the copy feature mimics the contents as shown.

**Comment by fred26** [comment]
Can this be implemented ?

**Comment by mrexodia** [comment]
Not really. To know the column width you have to know the maximum size of
the column before copying. It is possible to do, but it requires copying
the columns twice, which is very slow. Perhaps the option to copy only the
disassembly is more useful.

Note that I don't see the point in copying all of the module. What are you
trying to do exactly?

**Comment by fred26** [comment]
I am trying to copy the disassembly or part of the code section. The problem is that the function names and debug information gets all cropped/truncated no matter how big I resize the columns. The function's names are large as the classes' name are included as well.

**Comment by mrexodia** [comment]
I know that you are trying to copy part of the code section, but why?

**Comment by fred26** [comment]
I don't own IDA. I want to get the code in a text and try to do some analytics on top of it.

**Comment by mrexodia** [comment]
The purpose of x64dbg is to do in-memory analysis, I have no interest of
adding more functionality related to exporting assembly. There are demo and
free versions available for IDA that should do what you want.

If you have some specific kind of analysis in mind I recommend writing a
plugin that works directly on the disassembly, saving lots of parsing time
too.

**Comment by fred26** [comment]
Thanks so much.
Regards,
